### ISSUE TITLE
flatpak support

### ISSUE THREAD

**Issue by RustoMCSpit** [MeadowlarkDAW/Meadowlark#64]

**Comment by BillyDM** [comment]
Yeah, flatpak support is planned. Though of course it will still be a while until the first alpha release.

And it should be noted that because of the nature of flatpak's sandboxing, flatpak DAWs can only load plugins that were installed using flatpak. I would still recommend using a non-flatpak version of Meadowlark when the time comes if you can.
